src/components/JobFormS3.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Address from './Address';
+import { Form, Input, Label } from '../assets/styled-components/forms';
+
+class JobFormS3 extends React.Component {
+  render() {
+    return (
+      <section>
+        <h2>Job Info</h2>
+        <div>
+          ⊙====⊙====Ⓞ=---⊙----⊙
```

Comment:
The progress bar isn't quite halfway (see below)
⊙====⊙====Ⓞ=---⊙----⊙
⊙====⊙====Ⓞ==--⊙----⊙